examples/text_classification/README.md

```diff
@@ -1,7 +1,7 @@
 # This is an example to create a text classification dataset and train a sentiment model. 
 
 In the basic case, users can train the sentiment model in model.py with 
-AG_NEWS dataset in torchtext.datasets.text_classification. The dataset is
```

Comment:
Is this meant to indicate that the interface is in pre-release and will evolve?

Reply:
Good catch. We need to fix it in this release as all the text classification datasets are moved to `prototype` folder.

Reply:
@vincentqb - we talked a bit offline and thought it should be a good idea to move these new pieces into a preview or prototype folder first (even though we think they're much better) since the changes are quite drastic. What do you think?

Reply:
Are the previous datasets still being deprecated? Is the new interface finalized? 

If yes and yes, then why delay? :)

Reply:
The current datasets won't be deprecated during this release, instead we'll introduce a new prototype section.

The new interface is not finalized and we notice various inconsistencies bubbling up.

Reply:
`prototype` folder allows us to evolve the interface without a lot of pressure from BC breaking. We could improve the datasets API while receiving feedbacks from OSS users. Once they are matured, we could move them to the main folder.